docs/static/modules.asciidoc

```diff
@@ -110,6 +110,31 @@ For a list of available module settings, see the documentation for the module.
 [[overriding-logstash-module-settings]]
 ==== Specify module settings at the command line
 
+[IMPORTANT]
+.macOS Gatekeeper warnings
+====
+Apple's rollout of stricter notarization requirements affected the notarization
+of the {version} {ls} artifacts. If macOS Catalina displays a dialog when you
+first run {ls} that interrupts it, you will need to take an action to allow it
+to run.
+To prevent Gatekeeper checks on the {ls} files, run the following command on the
+downloaded `.tar.gz` archive or the directory to which was extracted:
+[source,sh]
+----
+xattr -d -r com.apple.quarantine <archive-or-directory>
+----
+For example, if the `.tar.gz` file was extracted to the default
+logstash-{version} directory, the command is:
+[source,sh,subs="attributes"]
+----
+xattr -d -r com.apple.quarantine logstash-{version}
+----
+Alternatively, you can add a security override for both `jdk.app` and
```

Comment:
Logstash does not have a `jdk.app` or `controller.app`. I'd suggest this language for this section:
```
Alternatively, you can add a security override if a Gatekeeper popup appears by following the instructions in the _How to open an app that hasn’t been notarized or is from an unidentified developer_ section of https://support.apple.com/en-us/HT202491[Safely open apps on your Mac].
```

Reply:
@Conky5 This is still a WIP, and your suggestion will help speed up the work. Thanks!